pkg/distributor/distributor.go

```diff
@@ -366,9 +366,9 @@ func (d *Distributor) checkSample(ctx context.Context, userID, cluster, replica
 // Validates a single series from a write request. Will remove labels if
 // any are configured to be dropped for the user ID.
 // Returns the validated series with it's labels/samples, and any error.
-func (d *Distributor) validateSeries(ts ingester_client.PreallocTimeseries, userID string) (client.PreallocTimeseries, error) {
+func (d *Distributor) validateSeries(ts ingester_client.PreallocTimeseries, userID string, skipLabelNameValidation bool) (ingester_client.PreallocTimeseries, error) {
```

Comment:
Nice catch! I've sent separate PR #3738 to fix double-import of `ingester_client` package in this file.

Reply:
We've just merged https://github.com/cortexproject/cortex/pull/3738. Could you rebase `master`, please?